acquia/acquia_cms
# ACMS-684: As a content editor, I want to be able to use Video using CKEditor correctly

**Motivation**

Fixes ACMS-684

**Proposed changes**

Update video media template
**Alternatives considered**


**Testing steps**


- Create article content in body field insert video, by clicking on "Insert from media library" select video tab and provide URL, click on add button and insert the media. 
- Check that video preview is available for embedded vide.
- Save the content, and check video is appearing on the page.

**Merge requirements**
